Bonsai/Bonsai.csproj

```diff
@@ -4,13 +4,18 @@
     <Description>A visual programming language for data stream processing built on top of Rx for .NET.</Description>
     <PackageTags>Bonsai Rx Reactive Extensions</PackageTags>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <TargetFrameworks Condition="'$(ContinuousIntegrationBuild)' != 'true'">net48;net6.0-windows</TargetFrameworks>
+    <TargetFramework Condition="'$(ContinuousIntegrationBuild)' == 'true'">net48</TargetFramework>
     <UseWindowsForms>true</UseWindowsForms>
-    <TargetFramework>net48</TargetFramework>
     <OutputType>Exe</OutputType>
     <DisableWinExeOutputInference>true</DisableWinExeOutputInference>
     <ApplicationIcon>..\Bonsai.Editor\Bonsai.ico</ApplicationIcon>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net48'">
```

Comment:
I think the recommendations now might be to use the application configuration settings in the `.csproj` file as showin in the WFAC010 page (unfortunately this Microsoft insists on not being consistent and changing their minds).

The only thing I can see that is not supported by the manifest file is changing the application default font and this might be why the recommendation started to change in the first place.

I will add a comment so we can remember to revisit this and anyway this might all change if we change to Dear ImGui.

Reply:
Interestingly we cannot change this specific condition to `TargetFrameworkIdentifier` otherwise the manifest does not embed correctly. It seems again that `TargetFrameworkIdentifier` does not bind "early enough" (all this magic bind order seems really hard to understand consistently).

Reply:
Interestingly we cannot change this specific condition to `TargetFrameworkIdentifier` otherwise the manifest does not embed correctly. It seems again that `TargetFrameworkIdentifier` does not bind "early enough" (all this magic bind order seems really hard to understand consistently).

Reply:
> does not bind "early enough" (all this magic bind order seems really hard to understand consistently).

That's my bad. It's actually due to a lack of magic that this doesn't work. `TargetFrameworkIdentifier` isn't assigned until [much later](https://github.com/dotnet/sdk/blob/323ad1ce32572096a231f63ea5a968509983899d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L49-L56) because it relies on `TargetFramework` which can come from the project so it has to happen in the `targets` stage. 

You need to manually extract the identifier from the `TargetFramework` instead. It's a little clunky but: `Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'"`

(Using `TargetFrameworkIdentifier` for targets is fine since they're not evaluated until they might run.)